backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -1,14 +1,22 @@
 """for each of the log entry creators in curriculum_tracking.activity_log_entry_creators, make sure it is called when it should be and creates the correct log entries
 """
 from rest_framework.test import APITestCase
-
+from django.test import TestCase
+import curriculum_tracking
 from test_mixins import APITestCaseMixin
 from . import factories
 from core.tests.factories import UserFactory
+from .factories import AgileCardFactory, RecruitProjectFactory, ProjectContentItemFactory
 from curriculum_tracking.models import AgileCard, ContentItem
 from activity_log.models import LogEntry
 from curriculum_tracking import activity_log_entry_creators as creators
+from curriculum_tracking import activity_log_entry_creators
```

Comment:
why are we importing these twice? look at line 12. Remember all the trouble that imports caused before? 

If something is already available then you probably dont need to import it again